.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,13 @@ https://petershaggynoble.github.io/MDI-Sandbox/simpleicons/preview/
 
 **Issue:**
 
+<!-- OR -->
```

Comment:
Would it be an option to just ask everyone to provide both an issue reference and Alexa rank (i.e. removing this comment)?

Reply:
Would save us one click on reviewing! So yes, I'd be in favor :)

Reply:
Before I _do_ update, see [this comment](https://github.com/simple-icons/simple-icons/issues/3200#issuecomment-642655069) and those that follow for the reasoning behind including the `<!-- OR -->`. If consensus still feels it's unnecessary, though, I'll remove it.

Reply:
If you are concerned about the Alexa rank already being reported in the issue, then I don't think it hurts to ask contributors to state it again in the PR (as it does indeed safe is one click and some potential search work, though it's usually in the first or second comment).

If you're concerned about another metric being more relevant, perhaps we can change the wording to "Popularity" or something similar?

Reply:
> If you are concerned about the Alexa rank already being reported in the issue, then I don't think it hurts to ask contributors to state it again in the PR
A good point: They can just copy/paste it from the issue - I was clearly overthinking things in that people would go and look it up again!

I've removed the `<!-- OR -->` now. I think we should just run with the  `**Alexa rank:**` label for now, until we've firmed away the cut-off points for some of our other metrics and added them to our guidelines.